zkvm/tests/zkvm.rs

```diff
@@ -0,0 +1,72 @@
+use bulletproofs::r1cs;
+use bulletproofs::{BulletproofGens, PedersenGens};
+use curve25519_dalek::ristretto::CompressedRistretto;
+use curve25519_dalek::scalar::Scalar;
+use merlin::Transcript;
+use std::collections::VecDeque;
+
+use zkvm::*;
+
+fn issue_contract(qty: u64, flv: Scalar, pred: &Predicate) -> Vec<Instruction> {
+    vec![
+        // pushdata qty [qty]
+        Instruction::Push(Commitment::from(CommitmentWitness::from_secret(qty)).into()),
+        // stack: qty-var
+        Instruction::Var,
+        // stack: flv, qty-var
+        Instruction::Push(Commitment::from(CommitmentWitness::unblinded(flv)).into()),
+        // stack: flv-var, qty-var
+        Instruction::Var,
+        // stack: pred, flv-var, qty-var
```

Comment:
These comments use right-to-left notation while the stack diagrams in the spec use left-to-right (top is on the right). Also would it be clearer if these comments were added on the same line as the instruction (since they denote the stack _after_ the instruction is executed)?

Reply:
ah, got it! I changed it to using left-is-top since that seemed to be how txvm worked (cc @cathieyun ), but can modify to have the top be on the right to align with the spec

Reply:
Oh, i probably didn't pay enough attention to internal txvm documentation. AFAIR, both TxVM and ZkVM specs use the same L-to-R stack diagram style, so i'd keep it the same in the code.

Reply:
oh i went back to the txvm docs and they are L-to-R, I misremembered! My bad, sorry vicki!